.gitignore

```diff
@@ -7,3 +7,5 @@ Gemfile.lock
 apps/**/translations/en
 coverage
 artefacts
+npm-shrinkwrap.json
```

Comment:
we should probably remove shrinkwrap

Reply:
It's removed in this PR

![screen shot 2017-11-15 at 11 58 00](https://user-images.githubusercontent.com/12494656/32834965-5a708d5c-c9fc-11e7-9071-bcfbbc7d0578.png)